spec/lib/lab_where_client_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LabWhereClient do
+  it 'works' do
+    expect(true).to be_truthy
+  end
+end
```

Comment:
Should probably delete this as its not actually testing anything.

Reply:
Ah. I added that file when I was going to add some tests for the client.